bower.json

```diff
@@ -28,13 +28,27 @@
     "maps"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.7.1",
-    "google-apis": "GoogleWebComponents/google-apis#^1.1.7",
-    "iron-resizable-behavior": "PolymerElements/iron-resizable-behavior#^1.0.5",
-    "iron-selector": "PolymerElements/iron-selector#^1.5.2"
+    "polymer": "Polymer/polymer#1.9 - 2",
+    "google-apis": "GoogleWebComponents/google-apis#2.0-preview",
```

Comment:
@e111077 Please change it to `google-apis#^2.0.0`. Recently `v2.0.0` of `<google-apis>` has been released: https://github.com/GoogleWebComponents/google-apis/releases/tag/v2.0.0.